clippy_lints/src/manual_float_methods.rs

```diff
@@ -90,10 +90,7 @@ impl<'tcx> LateLintPass<'tcx> for ManualFloatMethods {
             && let mut exprs = [lhs_lhs, lhs_rhs, rhs_lhs, rhs_rhs]
             && exprs.iter_mut().partition_in_place(|i| path_to_local(i).is_some()) == 2
             && !in_external_macro(cx.sess(), expr.span)
-            && (
-                matches!(cx.tcx.constness(cx.tcx.hir().enclosing_body_owner(expr.hir_id)), Constness::NotConst)
-                    || cx.tcx.features().declared(sym!(const_float_classify))
-            )
+            && matches!(cx.tcx.constness(cx.tcx.hir().enclosing_body_owner(expr.hir_id)), Constness::NotConst)
```

Comment:
If I understand the issue correctly, it sounds like we should rather replace the `cx.tcx.features().declared(...)` check with an MSRV check, since these functions are now stable in const.

With the current diff, we would never emit a warning in const code